deploy/operator.yaml

```
@@ -20,6 +20,18 @@ spec:
           command:
             - kubernetes-secret-generator
           imagePullPolicy: Always
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8080
+            initialDelaySeconds: 3
+            periodSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8080
+            initialDelaySeconds: 3
+            periodSeconds: 3
```

Maybe also expose and name the port?

```suggestion
          ports:
          - name: monitoring
            containerPort: 8080
          livenessProbe:
            httpGet:
              path: /healthz
              port: monitoring
            initialDelaySeconds: 3
            periodSeconds: 3
          readinessProbe:
            httpGet:
              path: /readyz
              port: monitoring
            initialDelaySeconds: 3
            periodSeconds: 3
```

---

Fixed, the deployment uses a named port now.